src/test/groovy/com/palantir/gradle/docker/PalantirDockerPluginTests.groovy

```diff
@@ -361,4 +361,34 @@ class PalantirDockerPluginTests extends AbstractPluginTest {
         execCond("docker rmi -f ${id}") || true
     }
 
+    def 'base image is pulled when "pull" parameter is set'() {
+        given:
+        String id = 'id8'
+        temporaryFolder.newFile('Dockerfile') << '''
+            FROM alpine:3.2
+        '''.stripIndent()
+        buildFile << """
+            plugins {
+                id 'com.palantir.docker'
+            }
+
+            dependencies {
```

Comment:
this block is irrelevant for this test; in particular, it doesn't pull the base image before building. please just pull the image explicitly before the test. (see DockerRunPlugin.groovy for how to do this with an Exec task)